BusinessLogic/PersonRecognizer.cs

```diff
@@ -86,14 +86,14 @@ public bool LoadTrainedFaces()
                 picturePath = ImageDataPath + directory + "/";
                 personLabelId++;
 
-                while (File.Exists(picturePath + picNumber + ".bmp"))
+                while (File.Exists(picturePath + picNumber + ConfigurationManager.AppSettings["PictureFormat"]))
```

Comment:
This could be extracted to a separate variable, instead of fetching it in three different places